### PR TITLE
Feature/ados598 personal info foreign fixes

### PR DESCRIPTION
**Pull request by Arochawms10**

### CODE REVIEW

File: src/app/screens/PersonalInformationForeign/styles.js [hunk 1]
```diff
@@ -10,12 +11,14 @@ export const Container = styled.View`
 export const TextWhite = styled.Text`
   color: ${(props) => props.theme.colors.white};
   align-self: center;
+  font-family: ${theme.fonts.SfProBold};
```
Comment (usaqui-albert) [inline src/app/screens/PersonalInformationForeign/styles.js:14]: In `TextWhite` you are using `theme` from props and from an import.

File: src/app/screens/PersonalInformationForeign/styles.js [hunk 2]
```diff
@@ -10,12 +11,14 @@ export const Container = styled.View`
...
 `;
 
 export const TextSmall = styled.Text`
   font-weight: ${({ weight = 'normal' }) => weight};
   color: ${(props) => props.theme.colors.white};
   align-self: center;
+  font-family: ${theme.fonts.SfProBold};
```
Comment (usaqui-albert) [inline src/app/screens/PersonalInformationForeign/styles.js:21]: In TextWhite you are using theme from props and from an import.

File: src/app/screens/PersonalInformationForeign/index.js [hunk 3]
```diff
@@ -1,5 +1,5 @@
 // @flow
-import React from 'react';
+import React, {useState} from 'react';
```
Comment (usaqui-albert) [inline src/app/screens/PersonalInformationForeign/index.js:2]: This line should raise an eslint error

File: src/app/screens/PersonalInformationForeign/styles.js [hunk 4]
```diff
@@ -29,12 +32,24 @@ export const IconSize = styled.Image`
 export const ContainerInput = styled.View`
   align-items: center;
   justify-content: center;
-  height: 50;
+  height: 55;
   width: 100%;
   padding-horizontal: 20;
   margin-bottom: 15;
+  position: relative;
```
Comment (jhonwms) [inline src/app/screens/PersonalInformationForeign/styles.js:39]: position
position in React Native is similar to regular CSS, but everything is set to relative by default, so absolute positioning is always relative to the parent.

**Pull request merged**